src/App.js

```diff
@@ -1,8 +1,43 @@
 import React from "react";
 import "./index.css";
+import Contacts from "./Components/Contacts";
+import ContactUserCard from "./Components/ContactUserCard"
+import NavBar from "./Components/NavBar";
+import Post from "./Components/Post"
+import Posts from "./Components/Posts";
+import UserProfile from "./Components/UserProfile";
 
 const App = () => {
-  return <p>Hello, world!</p>;
+  return (
+    <body className="App">
+      <header>
+      <NavBar>
```

Comment:
Hey Charlie, I wanted to write you a quick comment about how your components are written. I think that, by now after all the work we've done with React, that you probably see how they should be written. While you actually 'can' nest buttons and other elements inside of Components, this isn't the "react way" of doing things. Just for example, the buttons inside of your NavBar should be actually written inside of that component, not nested like HTML elements inside of the App component. You should only be displaying <NavBar /> inside of <App />, while everything else is written in the NavBar.js file.

In all of your individual components, you created a variable that returns a string and exported it (i.e. const Contacts = "Contacts";
export default Contacts; ) I have a feeling that you will see, or do see, by now that that is not a React component. If you need clarification on anything, please reach out to me, Sam, or Greg so we can help!